src/checkstyle/checkstyle.xml

```diff
@@ -438,8 +424,22 @@ page at http://checkstyle.sourceforge.net/config.html -->
             <property name="severity" value="error"/>
         </module>
 
-        <!-- Required to support SuppressWarningsComment -->
-        <module name="FileContentsHolder"/>
+    </module>
+
+    <!-- LENGTH CHECKS -->
+    <module name="LineLength">
```

Comment:
The `LineLength` module should be outside of the `TreeWalker` module.
See https://github.com/checkstyle/checkstyle/issues/2116 for details